tests/conftest.py

```diff
@@ -206,54 +206,37 @@ def webhook_change_status_assignee():
             "added": "mathieu@mozilla.com",
         },
     ]
-    event = webhook_event_factory(routing_key="bug.modify", changes=changes)
-    webhook_payload = webhook_factory(event=event)
+    event = factories.webhook_event_factory(routing_key="bug.modify", changes=changes)
+    webhook_payload = factories.webhook_factory(event=event)
     return webhook_payload
 
 
 @pytest.fixture
 def webhook_modify_private_example() -> BugzillaWebhookRequest:
-    event = webhook_event_factory(action="modify", routing_key="bug.modify:status")
-    webhook_payload = webhook_factory(
+    event = factories.webhook_event_factory(
+        action="modify", routing_key="bug.modify:status"
+    )
+    webhook_payload = factories.webhook_factory(
         bug={"id": 654321, "is_private": True}, event=event
     )
     return webhook_payload
 
 
 @pytest.fixture
-def action_example() -> Action:
-    return action_factory()
+def action_factory() -> Action:
+    return factories.action_factory
 
 
 @pytest.fixture
-def action_example_with_inner_match() -> Action:
-    return action_factory(whiteboard_tag="inner-match")
-
-
-@pytest.fixture
-def action_example_without_brackets_required() -> Action:
-    return action_factory(whiteboard_tag="bracket-less", brackets_required=False)
+def action_example() -> Action:
+    return factories.action_factory()
```

Comment:
_with the usage of `action_factory` from the `factories` module being imported to the namespace; the name `action_factory` was not possible in this file prior._

ref: https://github.com/mozilla/jira-bugzilla-integration/pull/472#discussion_r1167237261